src/modules/rating/services/rating-scheduler.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable, Logger } from '@nestjs/common'
+import { Cron } from '@nestjs/schedule'
+import * as dotenv from 'dotenv'
+import { RatingRepository } from '../database/rating.repository'
+
+@Injectable()
+export class RatingScheduler {
+	private readonly logger = new Logger(RatingScheduler.name)
+
+	constructor(private readonly ratingRepository: RatingRepository) {
+		dotenv.config()
+	}
+
+	@Cron(process.env.CRON_SCHEDULE)
```

Comment:
Don't use the environment variable directly from `process.env`.
To resolve this case, we need to initialize the cron job dynamically. Please checkout this document:
https://docs.nestjs.com/techniques/task-scheduling#dynamic-cron-jobs